feature_search/src/main/java/cm/aptoide/pt/feature_search/data/network/service/SearchRetrofitService.kt

```diff
@@ -0,0 +1,65 @@
+package cm.aptoide.pt.feature_search.data.network.service
+
+import cm.aptoide.pt.feature_apps.data.network.model.BaseV7DataListResponse
+import cm.aptoide.pt.feature_search.data.network.RemoteSearchRepository
+import cm.aptoide.pt.feature_search.data.network.model.SearchAppJsonList
+import cm.aptoide.pt.feature_search.data.network.model.TopSearchAppJsonList
+import cm.aptoide.pt.feature_search.data.network.response.SearchAutoCompleteSuggestionsResponse
+import cm.aptoide.pt.feature_search.di.RetrofitBuzz
+import cm.aptoide.pt.feature_search.di.RetrofitV7
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flowOf
+import retrofit2.Response
+import retrofit2.http.GET
+import retrofit2.http.Path
+import retrofit2.http.Query
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class SearchRetrofitService @Inject constructor(
+  @RetrofitBuzz private val autoCompleteSearchSuggestionsService: AutoCompleteSearchRetrofitService,
+  @RetrofitV7 private val searchAppRetrofitService: SearchAppRetrofitService
+) :
+  RemoteSearchRepository {
+
+  override suspend fun getAutoCompleteSuggestions(keyword: String): Response<SearchAutoCompleteSuggestionsResponse> {
+    return autoCompleteSearchSuggestionsService.getAutoCompleteSuggestions(keyword)
+  }
+
+  override suspend fun searchApp(keyword: String): Response<BaseV7DataListResponse<SearchAppJsonList>> {
+    return searchAppRetrofitService.searchApp(keyword, 15)
+  }
+
+  interface AutoCompleteSearchRetrofitService {
+    @GET("/v1/suggestion/app/{query}")
```

Comment:
v1 should probably be next to the baseUrl? what do you think? at least for the ws75.aptoide.com v7 we did that.

Reply:
Discussion: We decided we should have the version here, to make it easier to manage versions.